[DOC-75] Anchor link fixes and Maintenance copyedit v23.6 (#4063)
EventStore/EventStore: docs/operations.md

Scavenged chunks may be small enough to be merged into a single physical chunk file of approximately 256 MB. This behaviour can be disabled with this option.

| Format               | Syntax                                |
| :------------------- | :------------------------------------ |
| Command line         | `--disable-scavenge-merging`          |
| YAML                 | `DisableScavengeMerging`              |
| Environment variable | `EVENTSTORE_DISABLE_SCAVENGE_MERGING` |

**Default**: `false`, small scavenged chunks are merged together.

### Scavenge history

> Each scavenge operation gets an ID and creates a stream. You might want to look at these streams to see the

Reviewer: ```suggestion
Each scavenge operation is assigned a unique ID and creates a corresponding stream. These streams provide valuable insights into the scavenge history, including the duration of each operation and the amount of disk space reclaimed. However, retaining this history indefinitely may not be necessary. To manage this, you can limit the duration for which scavenge history is stored in the database using the option below:```

> scavenge history, how much time each operation took, and how much disk space was reclaimed. However, you

Reviewer: ```suggestion
```

> might not want to keep this history forever. Use the following option to limit how long the scavenge history

Reviewer: ```suggestion
```
